src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/index.js

```diff
@@ -30,12 +30,12 @@ const ApplicationRow = ({ applicationDetails, labelId }) => {
         </TableCell>
       </TableRow>
       <TableRow>
-        <TableCell className={'apartment-sub-table'} colSpan={5}>
+        <TableCell className={'collapsible-row'} colSpan={5}>
```

Comment:
They do it everywhere I have looked, and I don't know why though... 
I think it has something to do with whether it is in the JSX scope or in the Javascript code scope, since the `{...}` is for inline code, (inline with the HTML / JSX stuff). When working with raw HTML, I always see it at a normal string, but in the ReactJS stuff I see it always done this way, and I never bothered to research why

Reply:
Hm, that makes sense